jenkins/bootstrap.py

```diff
@@ -215,7 +215,15 @@ def checkout(call, repo, branch, pull, ssh=False):
         refs, checkouts = branch_ref(branch)
 
     git = 'git'
-    call([git, 'init', repo])
+    if git_cache:
```

Comment:
Actually I think this belongs under `IntegrationTest`. If I want to switch it to call `checkout` directly then I need to copy the line from `bootstrap` that deals with `_call`. It's not a big deal, but a little scary.